src/sagemaker/workflow/pipeline.py

```diff
@@ -75,7 +75,7 @@ def to_request(self) -> RequestType:
     def create(
         self,
         role_arn: str,
-        description: str = None,
+        pipeline_description: str = None,
```

Comment:
Any reason to change this beyond aesthetics? It will be a breaking change, and seems like there is [precedence](https://github.com/aws/sagemaker-python-sdk/blob/cc1932514250e9fc20bf55d38e8b2b57ffdf959e/src/sagemaker/lineage/context.py#L167) for the prior convention.

Reply:
docstring parameter string used is `pipeline_description`. I changed it to match it. This is not a breaking change. It passes all unit test

Reply:
Then the docstring needs to be changed. If someone is passing this with named arguments (`pipeline.create(description='My description', ...)`) they will get `create() got an unexpected keyword argument 'description'`. The unit tests pass because they're all using positional arguments.

Reply:
oh right forgot about that. I'll change the docstring